New version: OliverBetz.ExifTool version 13.10

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #200346

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

![Image](https://github.com/user-attachments/assets/7b228b23-137e-4d6a-9a52-544d417b6489)
![Image](https://github.com/user-attachments/assets/f4ce647f-71ee-4690-ad8d-87c9cdac2992)

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/200906)

Dragon1573 committed:
New version: OliverBetz.ExifTool version 13.10

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Dragon1573 commented on manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml:
```diff
@@ -0,0 +1,58 @@
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://oliverbetz.de/cms/files/Artikel/ExifTool-for-Windows/ExifTool_install_13.10_32.exe
+  InstallerSha256: EC1DE44FD23992F497626DE71FD4D41E122114450D52A58C7901C3E8E7EDC4D6
```
```suggestion
  InstallerSha256: EF2680E717A1B4C3729E931BE5C526FA1EECBA2068E77E960E74B9FD8C946036
```

Dragon1573 commented on manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml:
```diff
@@ -0,0 +1,58 @@
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://oliverbetz.de/cms/files/Artikel/ExifTool-for-Windows/ExifTool_install_13.10_32.exe
+  InstallerSha256: EC1DE44FD23992F497626DE71FD4D41E122114450D52A58C7901C3E8E7EDC4D6
```
```suggestion
  InstallerSha256: EF2680E717A1B4C3729E931BE5C526FA1EECBA2068E77E960E74B9FD8C946036
```

Dragon1573 reviewed (commented):
_Komac_ generated an incorrect manifest, installer file should be "re-hash".

Dragon1573 committed:
Apply suggestions from code review

- Correction of `InstallerSha256` of `x86` arch installer

mdanish-kh commented on manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml:
```diff
@@ -0,0 +1,58 @@
+UpgradeBehavior: uninstallPrevious
+ProductCode: ExifTool_is1
+ReleaseDate: 2024-12-20
+ElevationRequirement: elevatesSelf
```
```suggestion
```

`Scope: user` doesn't ask for elevation.

mdanish-kh commented on manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml:
```diff
@@ -0,0 +1,58 @@
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://oliverbetz.de/cms/files/Artikel/ExifTool-for-Windows/ExifTool_install_13.10_64.exe
+  InstallerSha256: EC1DE44FD23992F497626DE71FD4D41E122114450D52A58C7901C3E8E7EDC4D6
```
```suggestion
  InstallerSha256: EC1DE44FD23992F497626DE71FD4D41E122114450D52A58C7901C3E8E7EDC4D6
  ElevationRequirement: elevatesSelf
```

mdanish-kh commented on manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml:
```diff
@@ -0,0 +1,58 @@
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://oliverbetz.de/cms/files/Artikel/ExifTool-for-Windows/ExifTool_install_13.10_32.exe
+  InstallerSha256: EF2680E717A1B4C3729E931BE5C526FA1EECBA2068E77E960E74B9FD8C946036
```
```suggestion
  InstallerSha256: EF2680E717A1B4C3729E931BE5C526FA1EECBA2068E77E960E74B9FD8C946036
  ElevationRequirement: elevatesSelf
```

Dragon1573 committed:
Apply suggestions from code review

- Move elevation requirements to installer entrance level
- User scope does not need elevation

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>
